mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/server/master/MasterMain.java

```diff
@@ -346,20 +288,6 @@ public MasterConfiguration getConfig() {
         return config;
     }
 
-    public String getDescriptionJson() {
-        try {
-            return DefaultObjectMapper.getInstance().writeValueAsString(leadershipManager.getDescription());
-        } catch (JsonProcessingException e) {
-            throw new IllegalStateException(String.format("Failed to convert the description %s to JSON: %s", leadershipManager.getDescription(), e.getMessage()), e);
-        }
-    }
-
-    public Observable<MasterDescription> getMasterObservable() {
```

Comment:
looks like we still need to expose this from the new interface as we need it for some internal connections.

Reply:
"Observable<MasterDescription>" from the leader client.

Reply:
Just to be clear, do you mean expose this from `ILeadershipManager` or another interface?

Reply:
we still need "public Observable<MasterDescription> getMasterObservable()" method from teh masterMain class.

Reply:
ah, understood!  i'll push up a commit later today.

Reply:
@Andyz26 I pushed up a commit a few hours ago.

Also, I do not see anyway for me to approve snapshot creation.  I just see: "This branch is waiting to be deployed" with "Integrate Pull Request" with a link to the build.

Reply:
just approved. I guess there is another security setting somewhere I need to find.

Reply:
hey @Andyz26 - Just checking in to see if you found any more issues :)

Reply:
I got the control plane tested with some config changes and it seems to be working now. I will tried the agent/runtime parts today.